README.md

```diff
@@ -1,2 +1,15 @@
 # Bookmark-Telegram-Bot
 A telegram bot to organize and save useful links, mainly for twitter bookmark
+
+# Requirement
+- Node.js "https://nodejs.org/en/download"
+- grammY "https://grammy.dev"
+- @BotFather "https://telegram.me/BotFather"
```

Comment:
change `@BotFather "https://telegram.me/BotFather"` to telegram api key which can be obtained using @ BotFather. the focus is on telegram api key, not the botfather itself.